include/boost/geometry/algorithms/detail/overlay/get_clusters.hpp

```diff
@@ -0,0 +1,302 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2021 Barend Gehrels, Amsterdam, the Netherlands.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_ALGORITHMS_DETAIL_OVERLAY_GET_CLUSTERS_HPP
+#define BOOST_GEOMETRY_ALGORITHMS_DETAIL_OVERLAY_GET_CLUSTERS_HPP
+
+#include <algorithm>
+#include <vector>
+
+#include <boost/geometry/algorithms/detail/overlay/approximately_equals.hpp>
+#include <boost/geometry/algorithms/detail/overlay/cluster_info.hpp>
+#include <boost/geometry/algorithms/detail/overlay/get_ring.hpp>
+#include <boost/range/value_type.hpp>
+#include <boost/geometry/util/math.hpp>
+
+#define BOOST_GEOMETRY_USE_RESCALING_IN_GET_CLUSTERS
+
+
+namespace boost { namespace geometry
+{
+
+#ifndef DOXYGEN_NO_DETAIL
+namespace detail { namespace overlay
+{
+
+template <typename T>
+struct sweep_equal_policy
+{
+    template <typename P>
+    static inline bool equals(P const& p1, P const& p2)
+    {
+        // Using machine epsilon results in too much points considered as inequal
+        return approximately_equals(p1, p2, 1.0); // 1000
```

Comment:
I think it would be more intuitive if here a multiplier for machine epsilon was passed and there was no requirement on the caller's part to know that internally this number is multiplied by 1000 again before multiplying it by machine epsilon.

Reply:
The comment was left there by mistake

Reply:
My point was that I propose to pass the value of epsilon multiplier here, so (1000), instead of a multiplier for another multiplier which is unknown unless you actually see the `approximately_equals` function definition.

Reply:
:heavy_check_mark: done